### ISSUE TITLE
Error in Connection.py line 337

### ISSUE THREAD

**Issue by bereldhuin** [jcassel/OctoPrint-Siocontrol#5]
Hi,

In Connection.py, there is a bug at line 337 : 

```
                        elif line[:2] == "RR":  # IO ready for commands
                            self._logger.debug(f"IO claimed ready for commands:{line}")
                            self.enableCommandQueue = True

                        elif list[:2] == "IT":  # IO type List
                            self._logger.debug(f"IO Type list sent:{line}")

                        elif list[:2] == "DG":  # Debug Message
                            self._logger.debug(f"IO sent debug message:{line}")
```

The two variables `list` should be `line`

**Comment by jcassel** [comment]
Great catch. I will get an update out shortly.

**Comment by jcassel** [comment]
Corrected in release v0.6.4
